File: php/facadePattern/Flight.php
<?php  namespace php\facadePattern;
/*
|==========================================================================
|--------------------------------------------------------------------------
| 	The values for the properties would usually be obtained from a database
| 	Values have been explicitly added for example purposes.
|--------------------------------------------------------------------------
|==========================================================================
*/
class Flight{ 
  private $flightNo = 'A1W78783';
  private $airLine = 'PHPAir';
  private $depatureTime = '11-10-14 | 04:43';
  private $id;

	public function __construct($passID){
		$this->id = $passID;
	}
	
       /**
	* Get flight number
	* 
	* @return string
	*/
	public function getFlightNo(){
		
		return $this->flightNo;
	}
	
       /**
	* Get name of airline
	* 
	* @return string
	*/
	public function getAirLine(){
		
		return $this->airLine;
	}
	
       /**
	* Get flight departure time
	* 
	* @return string
	*/
	public function getDepatureTime(){
		
		return $this->depatureTime;
	}
}

File: java/facadePattern/Flight.java
package facadePattern;
/*
|========================================================================
|------------------------------------------------------------------------
| The values for the instance variables would 
|  usually be obtained from a database
|  However values have been added for the benfit of this example
|------------------------------------------------------------------------
|========================================================================
*/
public class Flight {
	private String id;
	private String flightNo = "AM676798";
	private String airLine = "Java-Air";
	private String departureTime = "12-11-14  10:43";

	public Flight(String id) {
		this.id = id;
	}

	public String getFlightNo() {
		return this.flightNo;
	}

	public String getAirline() {
		return this.airLine;
	}

	public String getDepartureTime() {
		return this.departureTime;
	}
}
